src/numberMask/index.js

```diff
@@ -1,20 +1,14 @@
-import * as R from 'ramda';
-
-// Removes non numeric values
-const removeNonNumeric = value => value.replace(/[^0-9.]+/g, '');
-
-const addCommasToNumber = (value = '') => value.replace(/(\d)(?=(\d\d\d)+(?!\d))/g, '$1,');
-
+/**
+ * Given a Number, return a masked version with commas and squash to 2 fractionals
+ * @param {Number} value Number to be masked
+ * @returns String
+ */
 const numberMask = (value) => {
-  const stringValue = String(value);
-  const isNegative = stringValue.charAt(0) === '-';
+  if (value === null || value === 0 || value === '') return 0;
```

Comment:
Can we just say

```
if (!value) { ... }
```

I feel if value is false or undefined, we want to also return 0